### PR TITLE
[TW-1515] Flows - Create with AI block

### PR DESCRIPTION
**Pull request by petewitcher-postman**
Adds an entry to the blocks list doc for the Create with AI block.

### CODE REVIEW

File: src/pages/docs/postman-flows/reference/blocks-list.md [hunk 1]
```diff
@@ -225,6 +225,20 @@ Runs all of the blocks that come after it once for each item in the input list u
 
 ## Action blocks
 
+### Create with AI
+
+<img alt="Create AI block" src="https://assets.postman.com/postman-docs/v10/create-with-ai-block-v10.jpg" width="295px"/>
+
+Transforms incoming data and generates text, images, or JSON objects using prompts and Artificial Intelligence (AI).
```
Comment (nerminamiller-postman) [inline src/pages/docs/postman-flows/reference/blocks-list.md:232]:
```suggestion
Transforms incoming data and generates text, images, or JSON objects using Artificial Intelligence (AI) prompts.
```

File: src/pages/docs/postman-flows/reference/blocks-list.md [hunk 2]
```diff
@@ -225,6 +225,20 @@ Runs all of the blocks that come after it once for each item in the input list u
...
+
+**variable** - This input accepts data from other blocks and enables you to assign it to a variable. Select the **Add data blocks** icon <img alt="Add data blocks icon" src="https://assets.postman.com/postman-docs/v10/icon-pin-collection-v10.14.0.jpg#icon" width="16px"> to add more variables as data blocks. The **Create with AI** block can reference these variables.
```
Comment (nerminamiller-postman) [inline src/pages/docs/postman-flows/reference/blocks-list.md:234]:
```suggestion
**variable** - Take data from other blocks and assign it to a variable. Select the **Add data blocks** icon <img alt="Add data blocks icon" src="https://assets.postman.com/postman-docs/v10/icon-pin-collection-v10.14.0.jpg#icon" width="16px"> to add more variables as data blocks. The **Create with AI** block can reference these variables.
```

File: src/pages/docs/postman-flows/reference/blocks-list.md [hunk 3]
```diff
@@ -225,6 +225,20 @@ Runs all of the blocks that come after it once for each item in the input list u
...
+
+**Mode selection dropdown list** - Select **Text**, **Image**, or **JSON** to specify the type of data you want to create. Select **Smart** to let the **Create with AI** block automatically recognize the type of data you are asking for.
```
Comment (nerminamiller-postman) [inline src/pages/docs/postman-flows/reference/blocks-list.md:236]:
```suggestion
**Mode selection dropdown list** - Select **Text**, **Image**, or **JSON** to specify the type of data you want to create. Select **Smart** to let the **Create with AI** block automatically recognize the type of data you need.
```

File: src/pages/docs/postman-flows/reference/blocks-list.md [hunk 4]
```diff
@@ -225,6 +225,20 @@ Runs all of the blocks that come after it once for each item in the input list u
...
+
+**Text entry block** - Enter and test AI prompts here.
```
Comment (nerminamiller-postman) [inline src/pages/docs/postman-flows/reference/blocks-list.md:238]:
```suggestion
**Text entry block** - Enter and test AI prompts.
```

File: src/pages/docs/postman-flows/reference/blocks-list.md [hunk 5]
```diff
@@ -225,6 +225,20 @@ Runs all of the blocks that come after it once for each item in the input list u
...
+
+**Output** - Sends data from the **Create with AI** block. The icon next to this port changes depending on which mode is selected. It shows **Aa** for Text, **?** for Image, and **()** for JSON.
```
Comment (nerminamiller-postman) [inline src/pages/docs/postman-flows/reference/blocks-list.md:240]:
```suggestion
**Output** - Send data from the **Create with AI** block. The icon next to this port changes depending on which mode is selected. It shows **Aa** for Text, **?** for Image, and **()** for JSON.
```

**Review by nerminamiller-postman — COMMENTED** [review]
Looks great, Pete! Added some suggestions for parallelism.

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Comment by rabbah** [comment]
is there a preview link (can we add preview links automatically to PR?)

**Review by rabbah — APPROVED** [review]

File: src/pages/docs/postman-flows/reference/create-with-ai.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+* Summarize the response in {{value1}}
+* List all the new customers in {{body}}
+* Visualize the data in {{expenses}} as a pie chart
+
+### The Create with AI block interface
+
+<img alt="Create AI block" src="https://assets.postman.com/postman-docs/v10/create-with-ai-block-v10.jpg" width="295px"/>
+
+**variable** - Take incoming data and assign it to a variable. Select the **Add data blocks** icon <img alt="Add data blocks icon" src="https://assets.postman.com/postman-docs/v10/icon-pin-collection-v10.14.0.jpg#icon" width="16px"> to add more variables as data blocks. Use curly braces to reference variables in your prompts like this: {{value1}}.
```
Comment (jkonrath-postman) [inline src/pages/docs/postman-flows/reference/create-with-ai.md:20]:
```suggestion
**variable** - Take incoming data and assign it to a variable. Select the **Add data blocks** icon <img alt="Add data blocks icon" src="https://assets.postman.com/postman-docs/v10/icon-pin-collection-v10.14.0.jpg#icon" width="16px"> to add more variables as data blocks. Use curly braces to reference variables in your prompts like this: `{{value1}}`.
```

**Review by jkonrath-postman — APPROVED** [review]
LGTM - spin this up locally and triple-check all the curly braces to make sure Gatsby doesn't eat them.

**Comment by petewitcher-postman** [comment]
> is there a preview link (can we add preview links automatically to PR?)

https://tw-1515-flows-ai-block.learning.postman-beta.com/docs/postman-flows/reference/create-with-ai/

**Pull request merged**